codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesresponse.java

```diff
@@ -494,218 +494,131 @@ public static Builder builder() {
-        }
-        if (other.structWithNestedTimestampMember() != null
-            && !other.structWithNestedTimestampMember().equals(this.structWithNestedTimestampMember())) {
-            return false;
-        }
-        if (other.blobArg() == null ^ this.blobArg() == null) {
-            return false;
-        }
-        if (other.blobArg() != null && !other.blobArg().equals(this.blobArg())) {
-            return false;
-        }
-        if (other.structWithNestedBlob() == null ^ this.structWithNestedBlob() == null) {
-            return false;
-        }
-        if (other.structWithNestedBlob() != null && !other.structWithNestedBlob().equals(this.structWithNestedBlob())) {
-            return false;
-        }
-        if (other.blobMap() == null ^ this.blobMap() == null) {
-            return false;
-        }
-        if (other.blobMap() != null && !other.blobMap().equals(this.blobMap())) {
-            return false;
-        }
-        if (other.listOfBlobs() == null ^ this.listOfBlobs() == null) {
-            return false;
-        }
-        if (other.listOfBlobs() != null && !other.listOfBlobs().equals(this.listOfBlobs())) {
-            return false;
-        }
-        if (other.recursiveStruct() == null ^ this.recursiveStruct() == null) {
-            return false;
-        }
-        if (other.recursiveStruct() != null && !other.recursiveStruct().equals(this.recursiveStruct())) {
-            return false;
-        }
-        if (other.polymorphicTypeWithSubTypes() == null ^ this.polymorphicTypeWithSubTypes() == null) {
-            return false;
-        }
-        if (other.polymorphicTypeWithSubTypes() != null
-            && !other.polymorphicTypeWithSubTypes().equals(this.polymorphicTypeWithSubTypes())) {
-            return false;
-        }
-        if (other.polymorphicTypeWithoutSubTypes() == null ^ this.polymorphicTypeWithoutSubTypes() == null) {
-            return false;
-        }
-        if (other.polymorphicTypeWithoutSubTypes() != null
-            && !other.polymorphicTypeWithoutSubTypes().equals(this.polymorphicTypeWithoutSubTypes())) {
-            return false;
-        }
-        if (other.enumTypeString() == null ^ this.enumTypeString() == null) {
-            return false;
-        }
-        if (other.enumTypeString() != null && !other.enumTypeString().equals(this.enumTypeString())) {
-            return false;
-        }
-        return true;
+  public boolean equals(Object obj) {
+    if (this == obj) {
+      return true;
+    }
+    if (obj == null) {
+      return false;
+    }
+    if (!(obj instanceof AllTypesResponse)) {
+      return false;
+    }
+    AllTypesResponse other = (AllTypesResponse) obj;
+    if (!Objects.equals(stringMember(), other.stringMember()) {
+      return false;
+    }
+    if (!Objects.equals(integerMember(), other.integerMember()) {
+      return false;
+    }
+    if (!Objects.equals(booleanMember(), other.booleanMember()) {
+      return false;
+    }
```

Comment:
Any chance we could just do return Object.equals() && Object.equals() & ...?

Reply:
+1 sounds good